### PR TITLE
O3-1359:Remove the unwanted 3.X test and workflows from QA framework

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose
The pull request is to remove the unwanted 3.x test workflows, test files, and fixture files from the QA framework and this pull request is related to the issue ticket number [O3-1359](https://issues.openmrs.org/browse/O3-1359)

## Goals
- Remove the tests which are not working since the issues related to the 3.X E2E testing framework.
- Update the guidance to a workable dockerized version.

## Approach
- Remove unwanted 3.x E2E tests cases and workflows.
- Remove the existing O3 docker file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,19 +125,37 @@ MySQL password should be the same for initialSetupTests as openmrs password
     ```
     npm install
     ```
+   
+## Setting up the local instance for the test backend.
```
Comment (jayasanka-sack) [inline README.md:129]:
```suggestion
## Setup up the local instance.
```

File: README.md [hunk 2]
```diff
@@ -125,19 +125,37 @@ MySQL password should be the same for initialSetupTests as openmrs password
...
-2. Run the app
+    mvn clean package
+    ```
+
+### If the build fails at this point should be destroyed the associated volumes
+```
+cd run/docker
+docker compose down -v
+mvn clean package
+```
```
Comment (jayasanka-sack) [inline README.md:147]:
```suggestion
* If the build fails, try deleting existing containers using (`docker compose down -v`)
```
cd run/docker
docker compose down -v
mvn clean package
```
```

Comment (pasindur99) [reply README.md:147]: @jayasanka-sack We should checkout to the docker directory to delete the existing container. So the cd `run/docker `command should be there, right?

Comment (jayasanka-sack) [reply README.md:147]: * If the build fails, navigate to the `run/docker` directory and try deleting existing containers using `docker compose down -v`

Comment (jayasanka-sack) [reply README.md:147]: Or maybe your original sugesstion: 

* If the build fails at this point should be destroyed the associated volumes
```
cd run/docker
docker compose down -v
```

File: README.md [hunk 3]
```diff
@@ -125,19 +125,35 @@ MySQL password should be the same for initialSetupTests as openmrs password
...
 
-1. Navigate to the docker folder
-    ``` 
-    cd docker
+2. Package the distribution and prepare the run
+    ```
+    mvn clean package
+    ```
+   
+* If the build fails, try deleting existing containers using (`docker compose down -v`)
```
Comment (jayasanka-sack) [inline README.md:142]:
```suggestion
* If the build fails, try deleting existing containers
```

File: README.md [hunk 4]
```diff
@@ -125,19 +125,35 @@ MySQL password should be the same for initialSetupTests as openmrs password
...
-    docker compose -f docker-compose-o3.yml up 
+    docker-compose up
     ```
-If the docker file won't up, follow the setup guidelines in [openmrs-distro-referenceapplication 3.x](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/3.x) branch.
 
+New OpenMRS UI is accessible at http://localhost/openmrs/spa
+
+OpenMRS Legacy UI is accessible at http://localhost/openmrs
+
+---
```
Comment (jayasanka-sack) [inline README.md:156]:
```suggestion
```

**Comment by pasindur99** [comment]
@jayasanka-sack fixed with [6adcf52](https://github.com/openmrs/openmrs-contrib-qaframework/pull/432/commits/6adcf52f27a9e52414e39b3fc4a9bc52b49c262a)

**Review by jayasanka-sack — APPROVED** [review]
Thanks @pasindur99 !

Merging this PR as it is independent of failing tests.

**Pull request merged**